lib/src/services/databases/dataRefresh/implementation.dart

```diff
@@ -0,0 +1,47 @@
+import "package:ramaz/models.dart";
+import "package:ramaz/services.dart";
+import "../../firestore.dart";
+import "interface.dart";
+///
```

Comment:
I know you're doing this because of the linter, but it's okay to have the linter yell at you here because it's right -- you DON'T want to merge this without any documentation. You can always open a PR with lint errors, you just can't merge it.